src/main.ts

```diff
@@ -311,6 +328,15 @@ export function prettifyExecutionGraphResult(executionGraphResult: Object): void
       `
     )
   )
+  if (testsTable.length > 1) {
+    core.summary
+      .addTable(testsTable)
+  } else if (vulnerabilitiesTable.length > 1) {
+    core.summary
+      .addTable(vulnerabilitiesTable)
+  }
```

Comment:
Note that these conditions aren't mutually exclusive:

```suggestion
  if (testsTable.length > 1) {
    core.summary.addTable(testsTable)
  }
  
  if (vulnerabilitiesTable.length > 1) {
    core.summary.addTable(vulnerabilitiesTable)
  }
```